core/unit_test/incremental/Test12a_ThreadScratch.hpp

```diff
@@ -120,9 +120,15 @@ struct ThreadScratch {
 
 TEST(TEST_CATEGORY, IncrTest_12a_ThreadScratch) {
   ThreadScratch<TEST_EXECSPACE> test;
+#ifdef KOKKOS_ENABLE_OPENMPTARGET
+  test.run(1, 32, 9);
+  test.run(2, 64, 22);
+  test.run(14, 128, 321);
+#else
```

Comment:
What is the reason for changing this?

Reply:
In the Release and RelWithDebInfo builds, the scratch implementation fails unless the team_size is a multiple of 32.

Reply:
Would you mind adding a FIXME with this comment (also in the other place)?